helm-chart/templates/deployment.yaml

```diff
@@ -57,3 +57,65 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app:  {{ template "json-rpc-relay.name" . }}-websocket
+    {{ include "json-rpc-relay.labels" . | nindent 4 }}
+  name: {{ include "json-rpc-relay.fullname" . }}-websocket
+  namespace: {{ include "json-rpc-relay.namespace" . }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.websocket.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "json-rpc-relay.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        app:  {{ template "json-rpc-relay.name" . }}-websocket
+        {{- include "json-rpc-relay.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "json-rpc-relay.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+      - name: {{ .Chart.Name }}-ws
+        image: "{{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        args: ["start:ws"]
+        env: 
+        - name: SUBSCRIPTIONS_ENABLED
+          value: "true" #SUBSCRIPTIONS_ENABLED set to "true" is a requirement for the websocket component
+        envFrom:
+        - configMapRef:
+            name: {{ include "json-rpc-relay.fullname" . }}
```

Comment:
Does this mean the JSON RPC API and the websocket deployment share the same config? SHouldn't they be different?
I'm wondering if instead we should have a separate chart?